wgpu-hal/src/dx12/command.rs

```diff
@@ -719,7 +724,8 @@ impl crate::CommandEncoder for super::CommandEncoder {
                         cat.clear_value.b as f32,
                         cat.clear_value.a as f32,
                     ];
-                    list.clear_render_target_view(*rtv, value, &[]);
+                    // TODO: Empty slice vs None?
+                    unsafe { list.ClearRenderTargetView(*rtv, &value, Some(&[])) };
```

Comment:
:eyes:

Reply:
I feel like `None` would be more obvious.

Reply:
Looks like the old `clear_render_target_view()` in `d3d12/src/command_list.rs` used to map an empty slice to `ptr::null()`, so I'm changing it to `None` too. If anything this is one of the few strange _semantic differences_ to perhaps have an effect on https://github.com/gfx-rs/wgpu/issues/6162?